### PR TITLE
(maint) Add a MAINTAINERS file

### PR DESCRIPTION
**Pull request by kylog**
Hi @hlindberg @joshcooper @MikaelSmith @thallgren @branan @HAIL9000 @Iristyle @whopper @adrienthebo @johnduarte @er0ck:

Anticipating some possible questions:

* This PR add a MAINTAINERS file to document who is maintaining this repo. The list of people started with the top 10 PR mergers in the last year, plus a little conversation from Michael and me here https://github.com/puppetlabs/kylo_scratchpad/pull/1/files#r76349506.
* If the list seems wrong, let's feel free to edit it.
* The format of the file is documented here: http://github.com/puppetlabs/maintainers
* And I used https://rubygems.org/gems/maintainers to construct the file
* I pushed the PR branch to the puppetlabs space (rather than mine) so that all parties can edit collaboratively as needed
* Note that this is a public repo. The email and name fields are *optional* here. As a starting point, I pre-populated them with the publicly available names/emails available for all of us on github (but took bogus or non-puppet emails as an indication that you may not want to share your puppet.com email). *Please feel free to edit your email and name fields for any reason.*
* This is just the second PR using this new MAINTAINERS format (because this repo gets more PRs than any other eng-maintained repo save ci-job-configs), and I'm sure there will be questions and prospects for improvement.

Thanks!

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "email": "eric.thompson@puppet.com",
+      "name": "Eric Thompson"
+    },
+    {
+      "github": "johnduarte",
+      "email": "john.duarte@puppet.com",
+      "name": "John Duarte"
+    },
+    {
+      "github": "adrienthebo",
```
Comment (MikaelSmith) [inline MAINTAINERS]: Missing an e-mail address?

**Comment by whopper** [comment]
👍

**Comment by johnduarte** [comment]
:+1:

**Comment by hlindberg** [comment]
👍

**Comment by HAIL9000** [comment]
:+1:

**Comment by er0ck** [comment]
👍

**Comment by branan** [comment]
Wait, why do Ethan and Adrien get away without having their emails listed? Can I too be part of this special club that cannot be held accountable?

**Comment by MikaelSmith** [comment]
Put up a new PR?

**Comment by kylog** [comment]
@branan I put in emails for people who had a valid at-puppet email address in their public github events stream (those two did not). But, the full name and the email address are both absolutely optional so please do put up a PR to remove your email.

Also, FYI I think that public event stream is populated by default from the .gitconfig from wherever you commit, so you might want to modify that as well.

**Comment by branan** [comment]
I'm totally fine having an email posted - that algorithm makes sense, and I will attempt to add more emoticons to my jokes in the future

**Comment by kylog** [comment]
Hah! I may have gotten a little humor-impaired and was also erring on the side of wearing my Privacy Advocate hat :)

**Comment by Iristyle** [comment]
Yeah I've always been a little paranoid to have any real email affiliated with my GitHub account, given open source repos are easily cloned and scraped. Any little bit that I can do to prevent more spam ;)

**Pull request merged**